File: btjs_page/to_work/old_btjs/structures/true_percentage.js



/*
Función que calcula el 
porcentaje de datos verdaderos 
de una lista booleana
*/
//return float
export function true_percentage(
        BOOL_ARRAY) {
    let true_number = 0
    for(let i in BOOL_ARRAY) {
        if(BOOL_ARRAY[i]) { 
            true_number += 1 
        }
    }
    let percent = true_number 
        * (100 / len(BOOL_ARRAY))
    percent = float(percent)
    return percent
}
    
    

File: pj_creator_2/btpy/btpy_list/mod/true_percentage/true_percentage.py



def true_percentage(ARRAY:list[bool])\
        ->float:
    """
    Función que calcula el 
    porcentaje de datos verdaderos 
    de una lista booleana
    """ 
    true_number = 0
    for e in ARRAY:
        if(e): true_number += 1
    percent = true_number * \
        (100 / len(ARRAY))
    percent = float(percent)
    return percent